cypress/page/menu-content.page.ts

```diff
@@ -1,9 +1,11 @@
 class MenuContentPage {
   private tShirtMenu: string;
+  private tDressMenu: string;
   private menuContentPageURL: string;
 
   constructor() {
     this.menuContentPageURL = "http://automationpractice.com/";
+    this.tDressMenu = "#block_top_menu > ul > li:nth-child(2) > a";
     this.tShirtMenu = "#block_top_menu > ul > li:nth-child(3) > a";
```

Comment:
you can improve these two locators to use title property in the a tag and removing the nth-child usage